pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -268,6 +256,8 @@ func (c *Cluster) removeMon(daemonName string) error {
 	delete(c.clusterInfo.Monitors, daemonName)
 	// check if a mapping exists for the mon
 	if _, ok := c.mapping.Node[daemonName]; ok {
+		// NOAH: this is not safe for unpinned monitors because they are
```

Comment:
mmm, i think that the health check will still remove a monitor that is not pinned if it appears to be unresponsive.

Reply:
Ah, I see now that you're referring to a mon that isn't pinned to the node. I was misinterpreting this. Is there still a TODO here though?

Reply:
yeh, it was a note about dereferencing what could be a nil pointer. this is obsolete since I just pushed a change that removes the `Port` field from mon/node mapping struct.